plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -50,11 +51,7 @@ public MavenAppEngineStandardWizardPage() {
     super("basicNewProjectPage"); //$NON-NLS-1$
     setTitle("Maven-based App Engine Standard Project");
     setDescription("Create new Maven-based App Engine Standard Project");
-    
-    // todo: pull this from commons
-    ImageDescriptor descriptor = AbstractUIPlugin.imageDescriptorFromPlugin(
-        "com.google.cloud.tools.eclipse.appengine.localserver", "icons/gcp-32x32.png"); //$NON-NLS-1$ //$NON-NLS-2$
-    this.setImageDescriptor(descriptor);
+    setImageDescriptor(AppEngineImages.googleCloudPlatform(32));
```

Comment:
> I'm not sure the provider should be responsible for figuring out what size is needed. The caller has that information, not the AppEngineImages class.

This code is requesting the wrong size for this situation.  I think the best we can right now is to describe the context for which the code is requested, and let the provider figure out the best image for that location.